ironfish/src/rpc/clients/client.ts

```diff
@@ -170,6 +171,10 @@ export abstract class RpcClient {
     ).waitForEnd()
   }
 
+  getAccountBalances(params: GetBalancesRequest): RpcResponse<void, GetBalancesResponse> {
```

Comment:
nit - should this be `getAccountsBalancesStream` to match the naming of the other streaming methods?

Reply:
Should this be streaming? It means HTTP adapter cannot use this down the line.